torch/_subclasses/fake_tensor.py

```diff
@@ -1608,6 +1617,9 @@ def _validate_output_for_cache_entry(
         kwargs: Mapping[str, object],
         output: Optional[FakeTensor],
     ) -> None:
+        if isinstance(output, (int, torch.SymInt, type(None))):
```

Comment:
should we give the same treatment here to unbacked symints? (check if any of these symints are unbacked and if so not cache hit)

Reply:
I think we will bail out before reaching this line. Wil add an assertion

Reply:
Added